cmake/sdk_versioning.cmake

```diff
@@ -0,0 +1,20 @@
+include_guard()
+
+function(obtain_project_version resultVar)
```

Comment:
resultVar allows caller to define whats the variable that will be overridden at the parent scope